data/spells/spells_us.txt

```diff
@@ -41,7 +41,7 @@
 43^Yaulp II^PLAYER_1^^^^You feel a surge of strength as you let forth a mighty yaulp.^ lets loose a mighty yaulp.^Your surge of strength fades.^0^0^0^0^500^2250^18000^8^3^0^15^0^30^20^-1^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^2518^2113^-1^-1^-1^-1^1^1^1^1^-1^-1^-1^-1^100^100^100^100^100^100^100^100^100^100^100^100^0^1^0^0^10^1^4^24^254^254^254^254^254^254^254^254^6^0^4^-1^0^0^255^19^39^255^255^255^255^255^255^255^255^255^255^255^255^255^43^0^0^7^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^44^88^0^0^0^0^0^0^0^0^0^0^0^7^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^5^101^20^92^8^4^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^1^1^0^0^0^0^0^-1^0^0^0^1^0^0^1^1^^0
 44^Yaulp III^PLAYER_1^^^^You feel a surge of strength as you let forth a mighty yaulp.^ lets loose a mighty yaulp.^Your surge of strength fades.^0^0^0^0^500^2250^12000^8^3^0^25^0^40^30^-7^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^2518^2113^-1^-1^-1^-1^1^1^1^1^-1^-1^-1^-1^100^100^100^100^100^100^100^100^100^100^100^100^0^1^0^0^10^1^4^24^254^254^254^254^254^254^254^254^6^0^4^-1^0^0^255^44^56^255^255^255^255^255^255^255^255^255^255^255^255^255^43^0^0^7^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^44^88^0^0^0^0^0^0^0^0^0^0^0^7^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^5^101^52^92^8^4^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^1^1^0^0^0^0^0^-1^0^0^0^1^0^0^1^1^^0
-45^Pacify^PLAYER_1^^^^You feel your aggression subside.^ looks less aggressive.^^200^0^0^0^3000^2250^6000^8^7^0^100^1^1^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^2504^2059^-1^-1^-1^-1^1^1^1^1^-1^-1^-1^-1^100^100^100^100^100^100^100^100^100^100^100^100^0^1^0^1^30^86^18^254^254^254^254^254^254^254^254^254^5^15^5^-1^0^0^255^39^51^255^255^255^255^255^255^255^255^255^255^39^255^255^44^0^0^12^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^100^0^40^87^0^15^0^0^0^0^0^0^0^0^0^11^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^5^101^49^98^8^7^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^1^1^0^0^0^0^0^-1^0^0^0^1^0^0^1^1^^0
+45^Pacify^PLAYER_1^^^^You feel your aggression subside.^ looks less aggressive.^^200^0^0^0^3000^2250^6000^8^70^0^100^1^1^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^2504^2059^-1^-1^-1^-1^1^1^1^1^-1^-1^-1^-1^100^100^100^100^100^100^100^100^100^100^100^100^0^1^0^1^30^86^18^254^254^254^254^254^254^254^254^254^5^15^5^-1^0^0^255^39^51^255^255^255^255^255^255^255^255^255^255^39^255^255^44^0^0^12^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^100^0^40^87^0^15^0^0^0^0^0^0^0^0^0^11^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^5^101^49^98^8^7^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^0^1^1^0^0^0^0^0^-1^0^0^0^1^0^0^1^1^^0
```

Comment:
I believe this was probably just a typo somehow, but I have confirmed that it is definitely supposed to last for duration 70, not 7.